www/source/demo/windows/steps/3.html.slim

```diff
@@ -0,0 +1,54 @@
+
+ p There is a third type of dependency, transitive dependencies, that do not need to be explicitly declared in your plan file. They are automatically included in the list of files when your package is built. See #{link_to 'Package contents','/docs/reference/package-contents/'} for more information.
+
+ h2 Where are the IIS and .NET Framework dependencies!?
+
+ p There are some dependencies in Windows that present unique challenges when trying to build a fully portable and isolatable package. The .NET framework and services like IIS are good examples of such dependencies. These represent monolithic, single tennant installations that cannot simply be downloaded to a specific place on disk and run in an isolated fashion. So we employ different patterns to deal with each of these.
+ 
+ h3 .NET Framework
+ 
+ p If this were a .NET Core application, we could simply declare a dependency on <code>core/dotnet-core</code> which would put <code>dotnet.exe</code> on our path and we just point that to our built binary and we are done. The .NET "full framework" on the other hand wants to be in a very specific place on disk, has a key component called the Gloabal Assembly Cache (note "global") and requires some Windows registry entries to be friendly. We will deal with this by leaving the responsibility of installing the correct version of the .NET framework up to our provisioning or configuration management layer. Our ASP.NET application plan will simply assume the presense of the .NET framework in the environment where it is running. The fact of the matter is that any fully patched version of Windows that Habitat supports will have a v4 .NET CLR installed - so often the requirement of the .NET frameork will already be present on a fresh Windows image. However it might be a version or two behind the latest version of the .NET 4 runtime and if your application depends on new features in those versions, then you will need to make sure to have them installed before running your application in Habitat.
+
+ h3 IIS
+
+ p The IIS binaries are actually baked into the Windows OS as a feature. Also, like the .NET framework, all web applications that run on a single node are managed by a single IIS instance. Rather than treating IIS as a traditional Habitat dependency, we will leverage #{link_to 'Powershell DSC (Desired State Configuration)','https://docs.microsoft.com/en-us/powershell/dsc/overview'} to enable and configure IIS to work with our application. This will be handled in our runtime hooks and we will discuss the details there.
+
+ h2 Use the Habitat Builder UI to search for dependencies
+
+ p By going to #{link_to 'the Habitat website',"#{builder_web_url}/#/pkgs/core"}, you can search for packages built by the Habitat team and members of the community, and use them in your own applications and services.
+
+ .screenshot
```

Comment:
I'd just delete this image reference (and the image), because this view no longer exists in Builder. If you want, you can replace it with this one:

<img width="1200" alt="image" src="https://user-images.githubusercontent.com/274700/41933398-9759862e-7938-11e8-95d5-95b654d2f37f.png">